register/views.py

```diff
@@ -16,8 +25,32 @@ def register(request):
         if user_form.is_valid():
             user = user_form.save()
             user.set_password(user.password)
+            user.is_active = False
             user.save()
-            return redirect(reverse('mainapp:home_page'))
+
+            # encoding user id to send over the network
+
+            uidb64 = urlsafe_base64_encode(force_bytes(user.pk))
+            domain = get_current_site(request).domain
+            link = reverse('register:activate', kwargs={'uidb64':uidb64, 'token':token_generator.make_token(user)})
+
+            activate_url = 'http://'+domain+link
+
+            email_subject = 'Activate your account'
+            email_body = 'Hi '+user.username+' Please use this link to verify your account.\n'+activate_url
+
+            email = EmailMessage(
+                email_subject,
+                email_body,
+                'testexpenses724@gmail.com',
+                [user.email],
+                )
+
+            email.send(fail_silently=False)
+            messages.success(request, 'Account successfully created')
```

Comment:
The message should be 'Account successfully created. Please check your email for the verification link'